docs/monthly-meeting/2020-09-17.md

```diff
@@ -0,0 +1,66 @@
+# JupyterHub and BinderHub Team Meeting - September
+
+- **Date:** Thursday 17th September, 2020
+- **Time:** 5pm UTC
+  - **Your timezone:** https://arewemeetingyet.com/UTC/2020-09-17/17:00/Hubs-Team-Meeting
+- **This HackMD:** https://hackmd.io/@sgibson91/hubs-team-meeting
+- **GitHub issue:** https://github.com/jupyterhub/team-compass/issues/335
+- **Calendar for future meetings:** https://jupyterhub-team-compass.readthedocs.io/en/latest/meetings.html
+
+## Welcome to the Team Meeting
+
+Hello!
+
+If you are joining the team video meeting, sign in below so we know who was here. Roll call:
+
+- name / institution / GitHub handle
+- Sharan Foga / None / sharanf
+- Sarah Gibson / The Alan Turing Institute / sgibson91
+- Tim Head / Binder / @betatim
+- Georgiana Dolocan / CIR / @GeorgianaElena
+- Richard Darst
+- Simon Li / OME, University of Dundee / @manics
+- Rollin Thomas / NERSC, LBL / @rcthomas
+- Yuvi Panda / UC Berkeley, 2i2c / @yuvipanda
+- Kevin Xu / The Alan Turing Institute / @kevinxufs
+
+## Quick updates
+
+60 second updates on things you have been up to, questions you have, or developments you think people should know about. Please add yourself, and if you do not have an update to share, you can pass.
+
+- Azure / Kubernetes API resiliency PR landed!
+  - <https://github.com/jupyterhub/kubespawner/pull/433>
+
+## Reports and celebrations
+
+This is a place to make announcements (without a need for discussion). This is also a great place to give shout-outs to contributors! We'll read through these at the beginning of the meeting.
+
+- None reported
+
+## Agenda items
+
+Let's collect all potential agenda items here before the start of the meeting. We will then attempt to create a coherent agenda that fits in the 60m meeting slot. If there are similar items try and group them together.
+
+- Sharan Foga (5 minutes): Community Statistics Proof of Concept for Jupyterhub
+  - Apache Kibble: anyone can deploy their own
+  - Get in touch with Sharan if you'd like to be added: sharan.foga@gamil.com
```

Comment:
Should `gamil.com` be `gmail.com`?
(Also should we omit the email from the minutes?)

Reply:
Yep, typo. And hmmmm, ethics in me is saying yes, but it was an open forum... I think let's do it and we can point folks in the right direction if anyone asks.